predict.py

```diff
@@ -0,0 +1,157 @@
+
+        self.transforms = {}
+        for key in model_paths.keys():
+            if key in ['ffhq_encode', 'ffhq_frontalize', 'toonify']:
+                self.transforms[key] = transforms.Compose([
+                    transforms.Resize((256, 256)),
+                    transforms.ToTensor(),
+                    transforms.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5])])
+            elif key == 'celebs_sketch_to_face':
+                self.transforms[key] = transforms.Compose([
+                    transforms.Resize((256, 256)),
+                    transforms.ToTensor()])
+            elif key == 'celebs_seg_to_face':
+                self.transforms[key] = transforms.Compose([
+                    transforms.Resize((256, 256)),
+                    augmentations.ToOneHot(n_classes=19),
+                    transforms.ToTensor()])
+            elif key == 'celebs_super_resolution':
+                self.transforms[key] = transforms.Compose([
+                    transforms.Resize((256, 256)),
+                    augmentations.BilinearResize(factors=[16]),
+                    transforms.Resize((256, 256)),
+                    transforms.ToTensor(),
+                    transforms.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5])])
+
+
+    @cog.input("model", type=str, options=["celebs_seg_to_face", "celebs_sketch_to_face", "ffhq_encode",
+                                                "ffhq_frontalize", "celebs_super_resolution", "toonify"],
+               help="choose model type")
+    @cog.input("image", type=Path, help="input facial image")
+    def predict(self, model, image):
+        opts = self.opts[model]
+        opts = Namespace(**opts)
+        pprint.pprint(opts)
+
+
+        net = pSp(opts)
+        net.eval()
+        net.cuda()
+        print('Model successfully loaded!')
+
+        original_image = Image.open(str(image))
+        if opts.label_nc == 0:
+            original_image = original_image.convert("RGB")
+        else:
+            original_image = original_image.convert("L")
+        original_image.resize((256, 256))
+
+        # Align Image
+        if model not in ["celebs_sketch_to_face", "celebs_seg_to_face"]:
+            input_image = run_alignment(str(image))
+        else:
+            input_image = original_image
+
+        img_transforms = self.transforms[model]
+        transformed_image = img_transforms(input_image)
+
+        if model in ["celebs_sketch_to_face", "celebs_seg_to_face"]:
+            latent_mask = [8, 9, 10, 11, 12, 13, 14, 15, 16, 17]
+        else:
+            latent_mask = None
+
+        with torch.no_grad():
+            result_image = run_on_batch(transformed_image.unsqueeze(0), net, latent_mask)[0]
+        input_vis_image = log_input_image(transformed_image, opts)
+        output_image = tensor2im(result_image)
+
+        if model == "celebs_super_resolution":
+            res = np.concatenate([np.array(input_vis_image.resize((256, 256))),
```

Comment:
If we don't resize `result_image`, then we'll probably need to resize the `input_vis_image` to `(1024, 1024)` for the shapes to match.